bin/setup_notifications.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+"""
+Script to set up notifcations table
+"""
+
+from sqlalchemy import create_engine
+from gdcdatamodel.models.notifications import Base
```

Comment:
Should use `sheepdog.models` instead of `gdcdatamodel.models`, and update requirements in `sheepdog.models` accordingly.